test/dummy/spec/models/visitor_spec.rb

```diff
@@ -8,4 +8,4 @@
 describe Visitor, type: :model do
   it { should have_many(:role_users).class_name("Goldencobra::RoleUser") }
   it { should have_many(:roles).through(:role_users).class_name("Goldencobra::Role") }
-end
+end
```

Comment:
I don't understand the diff. Is there a newline missing at the end of the file? (there is supposed to be one)

Reply:
yes, newline was missing

Reply:
If so, there are auto options for code editors to always include a new line at the end of a file on save.